rules/using-digital-asset-manager/rule.md

```diff
@@ -0,0 +1,98 @@
+---
+type: rule
+title: Video Data - Do you use a Digital Asset Manager (DAM) like Adobe Bridge
+  to manage a stock library
+uri: using-digital-asset-manager-for-stock
+authors:
+  - title: William Fardell
+    url: https://ssw.com.au/people/alumni/will/
+    img: https://ssw.com.au/people/static/Will-Greentree-Profile-5b47d1b6e1f91074e10546d28020eb19.jpg
+  - title: SSW RulesGPT 3.5 turbo
+    url: https://rulesgpt.ssw.com.au/
+    img: https://rulesgpt.ssw.com.au/images/chatgpt-icon.svg
+related:
+  - how-to-name-documents
+  - semantic-versioning
+  - organize-and-back-up-your-files
+created: 2023-08-09T02:20:28.506Z
+guid: 114547d6-0e8b-47aa-aee6-5e6b12b57212
+---
+## Why get a DAM?
+
+Trying to manage your large library of digital files can be pretty difficult using just Windows File Explorer or Mac's Finder as you are limited to folder structure and file names, which can lead to lost and abandoned assets due to ridged search functionality. When trying to preview your assets, your operating system may also lack support for previewing proprietary formats (E.G. Fujifilm's .RAF or Nikon's .NEF). A Good digital asset manager like Adobe Bridge solves this.
+
+while there are several digital asset managers on the market, each with its strengths and weaknesses, for the majority of small teams, independent creatives and the budget-conscious, Adobe Bridge is more than enough to handle the task.
+
+## Why do you need a stock library?
+
+When video editing, you might need a certain shot that was not captured on production due to oversite or budget constrictions. That is why editors turn to stock footage libraries like the following:
+
+* https://www.filmsupply.com
```

Comment:
Can we make these links friendlier for readers?

E.g.

from
`https://www.pexels.com`

To
`pexels.com`

Reply:
Done